docs/user-guide/systemrequirements-zosmf.md

```diff
@@ -17,13 +17,14 @@ Follow these steps described in this article to configure z/OSMF.
     ```
     /D IPLINFO
     ```
-  **Expected results:** part of the output contains the release, for example, `RELEASE z/OS 02.02.00`.
+  **Expected results:**  
+  Part of the output contains the release, for example, `RELEASE z/OS 02.02.00`.
 
 2. Configure z/OSMF.
 
-    z/OSMF is a base element of z/OS V2.2 and V2.3, so it is already installed. But it might not be configured and running on every z/OS V2.2 and V2.3 system.
+    z/OSMF is a base element of z/OS V2.2 and V2.3, so it is already installed. However, z/OSMF might not be configured and running on every z/OS V2.2 and V2.3 system.
```

Comment:
Maybe we can update versions of z/OSMF, those z/OS versions quite old, what do you think?

Reply:
@janan07 was asking the same thing above, I'm ok with it, but I'm not sure which ones we should delete